comps/comps-foreman-rhel7.xml

```diff
@@ -34,6 +34,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-redhat-cloud-services-frontend-components</packagereq>
```

Comment:
```suggestion
      <packagereq type="default">tfm-nodejs-redhat-cloud-services-frontend-components</packagereq>
```
(and then please sort again)

`nodejs-redhat-cloud-services-frontend-components` needs to be added to `comps-foreman-el8.xml`